2_adventures/001/amaliahern/06/fizz_buzz.rb

```diff
@@ -0,0 +1,16 @@
+DIVISIBLE_BY_3 = "fizz"
+DIVISIBLE_BY_5 = "buzz"
+
+def fizzbuzz(number)
+  result = ""
+  if(number % 3 == 0)
+    result = DIVISIBLE_BY_3 
+  elsif(number % 5 == 0)
+    result << DIVISIBLE_BY_5 
+  elsif
+    result = number
```

Comment:
You have an if, then elsif.  Walk through this logic and you will see where it is going wrong.